src/robot/session-manager.ts

```diff
@@ -32,7 +32,7 @@ export default class SessionManager {
 
   private get client() {
     const transport = this.deferredTransport();
-    return createPromiseClient(RobotService, transport);
+    return createClient(RobotService, transport);
```

Comment:
Why the change to `createClient`?

Reply:
`createPromiseClient` is deprecated in favor of plain old `createClient`